### PR TITLE
Extend DPDK validation

### PR DESCRIPTION
**Pull request by SchSeba**
This commit add two more validations to the DPDK suite

1. check the selinux - validate we are able to remove the hugepage file after a dpdk run
2. run on multiple vendor - validate dpdk is working on multiple vendor

Signed-off-by: Sebastian Sch <sebassch@gmail.com>

### CODE REVIEW

File: cnf-tests/testsuites/e2esuite/dpdk/dpdk.go [hunk 1]
```diff
@@ -329,6 +341,69 @@ var _ = Describe("dpdk", func() {
...
+				Skip("Split VF test disabled for discovery mode")
+			}
+
+			CleanSriov()
+		})
+
+
+		DescribeTable("Test connectivity using the requested nic", func(vendorID, deviceID string) {
+			By("searching for the requested network card")
+			node, sriovDevice,exist := findSriovDeviceForDPDK(sriovInfos,nodeNames,vendorID, deviceID)
```
Comment (fedepaol) [inline cnf-tests/testsuites/e2esuite/dpdk/dpdk.go:372]: nit: missing some spaces in the arg list here

File: cnf-tests/testsuites/e2esuite/dpdk/dpdk.go [hunk 2]
```diff
@@ -329,6 +341,69 @@ var _ = Describe("dpdk", func() {
...
+				5,
+				false)
+			Expect(err).ToNot(HaveOccurred())
+
+			By("Parsing output from the DPDK application")
+			Eventually(func() string {
+				out, err = pods.GetLog(txOnlydpdkWorkloadPod)
+				Expect(err).ToNot(HaveOccurred())
+				return out
+			}, 8*time.Minute, 1*time.Second).Should(ContainSubstring(LOG_ENTRY),
```
Comment (fedepaol) [inline cnf-tests/testsuites/e2esuite/dpdk/dpdk.go:396]: Is this right? The test passes if the string is found in the logs?

**Review by fedepaol — COMMENTED** [review]
A question and a nit, otherwise looks good

File: cnf-tests/testsuites/e2esuite/dpdk/dpdk.go (hunk 1 above)
Comment (SchSeba) [reply cnf-tests/testsuites/e2esuite/dpdk/dpdk.go]: done

File: cnf-tests/testsuites/e2esuite/dpdk/dpdk.go (hunk 2 above)
Comment (SchSeba) [reply cnf-tests/testsuites/e2esuite/dpdk/dpdk.go:395]: then we ahve the `checkTxOnly(out)` in line 398

**Pull request merged**